student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './person'
+
+class Student < Person
+  attr_accessor @classroom
+
+  def initialize(classroom)
+    super
```

Comment:
- [ OPTIONAL] Add `(age, name, parent_permission)` to avoid confusion and to know better what are you inheriting here